xcode/Subconscious/Shared/Models/RecoveryPhrase.swift

```diff
@@ -11,26 +11,17 @@ import Foundation
 public struct RecoveryPhrase:
     Hashable,
     Equatable,
-    Identifiable,
-    Codable,
-    LosslessStringConvertible,
-    CustomDebugStringConvertible {
-    
-    public var debugDescription: String {
-        "<recovery phrase>"
-    }
-    
+    Codable
+{
     private static let recoveryPhraseRegex = /^(?:\w+\s+){23}\w+$/
 
-    public let description: String
-    public let verbatim: String
-    public var id: String { description }
+    // !!!: Mnemonic is secret and should never be logged or persisted
+    @Redacted public private(set) var mnemonic: String
```

Comment:
Create an explicit field for the secret mnemonic and redact it. This ensures you must access the property explicitly.

Calling `String(describing: recoveryPhrase)` will not accidentally reveal the mnemonic.

Reply:
Nice.